src/providers/settings.tsx

```diff
@@ -108,22 +108,37 @@ export const SettingsProvider: FC<SettingsProviderProps> = ({children}) => {
         apiSettings = {};
       }
 
-      const appConfig = Object.assign({}, defaultValue.appConfig, {
-        checkInMaxAge: Number(apiSettings.checkInMaxAge),
-        riskGroupMinAge: Number(apiSettings.riskGroupMinAge),
-        hsePhoneNumber: apiSettings.hsePhoneNumber
-      });
+      const appConfig: AppConfig = {...defaultValue.appConfig};
```

Comment:
Super minor, but as a guard against potential performance issues elsewhere, it would be good to still predefine these properties.
```suggestion
      const appConfig: AppConfig = {
        ...defaultValue.appConfig,
        checkInMaxAge: undefined,
        riskGroupMinAge: undefined,
        hsePhoneNumber: undefined
      };
```

Same with the traceConfiguration below